src/gameClasses/components/script/VariableComponent.js

```diff
@@ -594,10 +594,9 @@ var VariableComponent = IgeEntity.extend({
 
 				case 'stringToNumber':
 					var value = self.getValue(text.value, vars);
-					var parsedValue = parseFloat(value);
+					var parsedValue = Number(value.replace(' ', '')); //convenience measure
```

Comment:
this will throw errors if `value` is not `string`.

have you considered:
```javascript
var value = self.getValue(text.value, vars);
var parsedValue = Number(value);

if (!isNaN(parsedValue)) {
	returnValue = parsedValue;
}
```